### ISSUE TITLE
Not able to use GSLB examples as is

### ISSUE THREAD

**Issue by stephanerenou** [F5Networks/f5-appsvcs-extension#449]
### Environment
 * Application Services Version: 3.25
 * BIG-IP Version: BIG-IP 15.1.2.1 Build 0.0.10 Point Release 1

### Summary
Trying to use the GSLB examples from: https://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/latest/declarations/gslb.html#using-gslb-features-in-a-declaration
getting errors. First one as:  "\\\"last-resort-pool\\\" requires a value"
which can be fixed adding  "lastResortPool": {"use": "testPool" },
then next error is: Cannot read property 'toLowerCase' of undefined.

### Steps To Reproduce
1. Submit the following declaration:
```json
{
    "$schema": "https://raw.githubusercontent.com/F5Networks/f5-appsvcs-extension/master/schema/latest/as3-schema.json",
    "class": "AS3",
    "action": "deploy",
    "persist": true,
    "declaration": {
            "class": "ADC",
            "schemaVersion": "3.6.0",
            "id": "GSLB_Sample",
            "Sample_non_http_05": {
                "class": "Tenant",
                "Application": {
                    "class": "Application",
                    "testDomain": {
                        "class": "GSLB_Domain",
                        "domainName": "example.edu",
                        "aliases": [
                                      "another.example*" ],
                        "resourceRecordType": "A",
                        "poolLbMode": "ratio",
                        "pools": [
                                      { "use": "testPool" }
                        ]
                    },
                    "testPool": {
                       "class": "GSLB_Pool",
                        "enabled": false,
                        "lbModeAlternate": "ratio",
                        "lbModeFallback": "ratio",
                        "manualResumeEnabled": true,
                        "verifyMemberEnabled": false,
                        "qosHitRatio": 10,
                        "qosHops": 11,
                        "qosKbps": 8,
                        "qosLinkCapacity": 35,
                        "qosPacketRate": 5,
                        "qosRoundTripTime": 75,
                        "qosTopology": 3,
                        "qosVirtualServerCapacity": 2,
                        "qosVirtualServerScore": 1,
                        "members": [
                            {
                                "ratio": 10,
                                "server": {
                                    "use": "/Common/Shared/testServer"
                                },
                                "virtualServer": "0"
                            }
                        ],
                        "bpsLimit": 5,
                        "bpsLimitEnabled": true,
                        "ppsLimit": 4,
                        "ppsLimitEnabled": true,
                        "connectionsLimit": 3,
                        "connectionsLimitEnabled": true,
                        "maxAnswersReturned": 10,
                        "monitors": [
                            {
                                "bigip": "/Common/http"
                            },
                            {
                                "bigip": "/Common/https"
                            }
                        ],
                        "resourceRecordType": "A",
                        "fallbackIP": "1.1.1.1"
                    }
                }
            },
            "Common": {
                "class": "Tenant",
                "Shared": {
                    "class": "Application",
                    "template": "shared",
                    "testDataCenter": {
                        "class": "GSLB_Data_Center"
                    },
                    "testServer": {
                        "class": "GSLB_Server",
                        "dataCenter": {
                            "use": "testDataCenter"
                        },
                        "devices": [
                            {
                                "address": "1.2.3.7"
                            }
                        ],
                        "virtualServers": [
                            {
                                "address": "1.2.3.8",
                                "port": 5050
                            },
                            {
                                "address": "2001:0db8:85a3:0000:0000:8a2e:0370:7334",
                                "port": 5051
                            }
                        ]
                    }
        
                }
            }
        
    }
  }
```

2. Observe the following error response:
```json
               {
            "message": "Cannot read property 'toLowerCase' of undefined",
            "host": "localhost",
            "tenant": "Sample_non_http_07",
            "code": 422
        },
```

### Expected Behavior
a success return code, or at least an error message with more information

### Actual Behavior
nothing gets created in the tenant, and error message does not allow to understand the root cause.

**Comment by jmcalalang** [comment]
This error is related to: https://cdn.f5.com/product/bugtracker/ID1011285.html 

The internal issue is being worked on right now.

**Comment by stephanerenou** [comment]
Thanks for the feedback. I saw that other issue, but as here in the case of the example, lastResortPool was not empty, I was not sure this was identical and so could not use the workaround.
And then even defining the lastResortPool information, I could not understand the second issue and could not find a case related to that.
But good to hear that all those are pointing to the same issue being worked on as I don't see a current workaround to push the definition currently.

**Comment by stephanerenou** [comment]
Would you be closing this issue when the fix would be made available, or how can we get informed about the status?

**Comment by dstokesf5** [comment]
It looks like this was fixed in AS3 version 3.26.0.
